.github/workflows/check-attributions.yml

```diff
@@ -0,0 +1,31 @@
+name: Check Attributions
+
+on:
+  push:
+    branches: Version-v*
+  pull_request:
+    branches: Version-v*
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  check-attributions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - run: corepack enable
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+      - name: Install dependencies from cache
+        run: yarn --immutable
+      - name: Generate Attributions
+        run: yarn attributions:generate
```

Comment:
It looks like `yarn attributions:check` already runs `yarn attributions:generate`, so is this step necessary?

``` suggestion
```